mhcflurry/class1_affinity_prediction/class1_affinity_predictor.py

```diff
@@ -119,6 +119,27 @@ def supported_peptide_lengths(self):
             max(lower for (lower, upper) in length_ranges),
             min(upper for (lower, upper) in length_ranges))
 
+    def merge(self, affinity_predictors=[]):
+        """ 
+        Merge multiple affinity predictors into one.  
+        When training MHCFlurry in parallel, the user might be left with
+        multiple affinity predictors, merge can take in multiple affinity predictors
+        and the result will be a single ensemble.
+        Parameters
+        ----------
+        affinity_predictors : array
+            Affinity predictors that want to merge
+        """
+        self.allele_to_allele_specific_models
+        for affinity_predictor in affinity_predictors:
+            for allele in affinity_predictor.allele_to_allele_specific_models.keys():
+                if allele in self.allele_to_allele_specific_models:
+                    self.allele_to_allele_specific_models[allele] += affinity_predictor.allele_to_allele_specific_models[allele]
+                else:
+                    self.allele_to_allele_specific_models[allele] = []
+                    self.allele_to_allele_specific_models[allele] += affinity_predictor.allele_to_allele_specific_models[allele]
+            self.manifest_df = pandas.merge(self.manifest_df, affinity_predictor.manifest_df, on=['model_name', 'allele', 'config_json', 'model'], how='outer')
```

Comment:
hmm I'm not sure this does what we want. I would have thought `pandas.concat([... data frames ...], ignore_index=True` is what we want, not merge (which does a join)